testsuite/test-common.sh

```diff
@@ -45,6 +45,14 @@ hbasehadoop27zookeeper34java17versions_javaversion=${java17}
 hbase_test_groups="hbasehadoop22zookeeper34java16versions hbasehadoop27zookeeper34java17versions"
 hbase_all_versions="${hbasehadoop22zookeeper34java16versions} ${hbasehadoop27zookeeper34java17versions}"
 
+hivehadoop27zookeeper34java17versions="2.3.0"
+hivehadoop27zookeeper34java17versions_hadoopversion="2.7.3"
```

Comment:
for consistency to other parts in the test suite, could we go with hadoop 2.7.0 and zookeeper 3.4.11.